nkpremices/Banka
# #165163723 Test framework set up, set continuous integration (Travis CI, Heroku)

#### What does this PR do?
Have the server side configured

#### Description of Task to be completed?
* Setup the testing framework
* Write a simple test for a welcome message
* Setup travis integration 
* Create an app on heroku
* Setup continuous integration
* Add badges on the readme

#### How should this be manually tested?
* Go to the [github](https://github.com/nkpremices/Banka/) to see the badges
* With postmanm request a get on the heroku [link](https://banka-heroku.herokuapp.com/)
